tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/basic/api/ScalarTestApi.java

```diff
@@ -30,7 +32,7 @@
  */
 @GraphQLApi
 public class ScalarTestApi {
-    @Query
+    @Query("testScalarsInPojo")
```

Comment:
This change requires a change to `basicScalar/input.graphql` and `basicScalar/output.json` since the query name has changed from `scalarHolder` to `testScalarsInPojo`.

Reply:
Yes. Thanks. Fixed. Will be included in the next commit.